stable/database/values.yaml

```diff
@@ -402,6 +402,37 @@ database:
   isManualVolumeProvisioning: false
   isRestore: false
 
+  # NuoDB supports explicit database protocol version upgrade which is performed
+  # after upgrading the NuoDB image for all database processes. To simplify
+  # NuoDB version rollout, Kubernetes Aware Admin (KAA) is used to automatically
+  # upgrade database protocol and restart a Transaction Engine (TE) as an
+  # upgrade finalization step. For more information on how to perform the steps
+  # manually, please check Upgrading the Database Protocol
+  # (https://doc.nuodb.com/nuodb/latest/deployment-models/physical-or-vmware-environments-with-nuodb-admin/installing-nuodb/upgrading-to-a-new-release/upgrading-the-database-protocol/).
+  #
+  # The database protocol version is eligible for an upgrade to the max possible
+  # version if all of the bellows are true:
```

Comment:
I don't think we mention anywhere in the repo about the upgrade process itself (as in helm upgrade...)? Should we at least point out the minimum engine numbers required for a rolling upgrade somewhere?  Perhaps it is a seperate task to create a HowTo.

Reply:
Maybe now is a good time to document the upgrade process using Helm. I've created a separate DOC task for the _HowTo_ document.